pom.xml

```diff
@@ -237,7 +237,6 @@
             <bundledSignatures>
               <bundledSignature>jdk-unsafe</bundledSignature>
               <bundledSignature>jdk-deprecated</bundledSignature>
-              <bundledSignature>jdk-non-portable</bundledSignature>
```

Comment:
JFR annotations would lead to errors because they're non-portable. See https://github.com/policeman-tools/forbidden-apis/issues/234

Given that we bundle the JDK this is a non-issue and we can remove the forbidden signatures.